ui/src/components/Table.css

```diff
@@ -0,0 +1,12 @@
+.apps-table {
```

Comment:
@laurafitzgerald can we increase the size of the table head font?

Reply:
@damienomurchu sure. Should I make it lower case then?

Reply:
I added some styling to the table header and I think it makes it stand out now without increasing the font.